app/app/explore/LiquidityPools.tsx

```diff
@@ -0,0 +1,191 @@
+}
+
+const LiquidityPools: React.FC<LiquidityPoolsProps> = ({
+  pools,
+  runePriceUSD,
+}) => {
+  const [sortConfig, setSortConfig] = useState<SortConfig>({
+    key: SortKey.TVL,
+    direction: SortDirection.DESC,
+  });
+
+  const sortedPools = useMemo(() => {
+    const sortableItems = [...pools];
+    sortableItems.sort((a, b) => {
+      if (sortConfig.key === SortKey.TVL) {
+        const aTvl = calculatePoolTVL(a, runePriceUSD);
+        const bTvl = calculatePoolTVL(b, runePriceUSD);
+        return sortConfig.direction === SortDirection.ASC
+          ? aTvl - bTvl
+          : bTvl - aTvl;
+      } else if (sortConfig.key === SortKey.APR) {
+        const aApr = parseFloat(a.poolAPY);
+        const bApr = parseFloat(b.poolAPY);
+        return sortConfig.direction === SortDirection.ASC
+          ? aApr - bApr
+          : bApr - aApr;
+      }
+      return 0;
+    });
+    return sortableItems;
+  }, [pools, sortConfig, runePriceUSD]);
+
+  const getAssetSymbol = (asset: string) => {
+    return asset.split("-")[0] || asset;
+  };
+
+  const getLogoPath = (asset: string) => {
+    const assetLower = asset.toLowerCase();
+    return `https://storage.googleapis.com/token-list-swapkit-dev/images/${assetLower}.png`;
+  };
+
+  const calculateVolumeUSD = (pool: PoolDetail) => {
+    const volumeInRune = parseFloat(pool.volume24h) / 1e8;
+    return volumeInRune * runePriceUSD;
+  };
+
+  const calculateVolumeDepthRatio = (pool: PoolDetail) => {
+    const volumeUSD = calculateVolumeUSD(pool);
+    const tvlUSD = calculatePoolTVL(pool, runePriceUSD);
+    return volumeUSD / tvlUSD;
+  };
+
+  const sortData = (key: SortKey) => {
+    setSortConfig((prevConfig) => ({
+      key,
+      direction:
+        prevConfig.key === key && prevConfig.direction === SortDirection.ASC
+          ? SortDirection.DESC
+          : SortDirection.ASC,
+    }));
+  };
+
+  const topPools = sortedPools.slice(0, 3);
+  const topPoolsData = topPools.map((pool) => ({
+    asset: pool.asset,
+    formattedTVL: getFormattedPoolTVL(pool, runePriceUSD),
+    apr: parseFloat(pool.poolAPY),
+  }));
+
+  return (
+    <>
+      <TopCards
+        items={topPoolsData}
+        getAssetSymbol={getAssetSymbol}
+        getLogoPath={getLogoPath}
+      />
+
+      {/* All Pools table */}
+      <h2 className="text-xl font-medium mb-4">All Pools</h2>
+      <div className="overflow-hidden">
+        <div className="min-w-full">
+          <div className="flex text-left text-base text-gray-700">
+            <div className="px-3 py-3 w-1/2">Asset</div>
+            <div className="flex flex-1 w-1/2 justify-between">
+              <div className="px-3 py-3 w-1/4 ml-6">Volume (24h)</div>
+              <div className="px-3 py-3 w-1/4">Volume/Depth</div>
+              <div
+                className="px-3 py-3 w-1/4 flex items-center cursor-pointer"
+                onClick={() => sortData(SortKey.TVL)}
+              >
+                TVL
+                <Image
```

Comment:
another opinion - there's no indication which sort option is active. imo i think this icon should be present if it's column sorting is active. we should also bolden the top or bottom arrow when sorting desc or asc respectively.

can worry about this later.